File: rotatearray/one.cpp
#include <bits/stdc++.h>
using namespace std;

void rota(int arr[], int co, int len) {
    int th = 0;

    while (th < co) {
        int temp = arr[len-1];

        for (int i = 0; i < len; i++) {
            arr[len-i] = arr[len-1-i];
        }

        arr[0] = temp;
        th++;
    }

    for (int i = 0; i < len; i++) {
        cout << arr[i] << " ";
    }

    cout << endl;
}

int main() {
    int arr[] = {-1,-100,3,99};
    int len = sizeof(arr)/sizeof(arr[0]);

    rota(arr, 2, len);

}

File: rotatearray/one.java
package rotatearray;
import java.util.*;
public class one {
    public static void main(String[] args) {
        int[] arr = {1,2,3,4,5,6,7};
        int n = arr.length;
        int co = 0;
        while (co < 2) {
            int temp = arr[n-1];
            for (int i = 0; i < n-1; i++) {
                arr[n-i] = arr[n-1-i];
            }
            arr[0] = temp;
            co++;
        }

        System.out.println(Arrays.toString(arr));

    }
}